### PR TITLE
checkpoint-sync nit, go isn't required on source node

### PR DESCRIPTION
**Pull request by muang0**
#594

### CODE REVIEW

File: website/docs/prysm-usage/checkpoint-sync.md [hunk 1]
```diff
@@ -85,6 +79,12 @@ Installing `prysmctl` via `prysm.sh`, or downloading it from prysm's github rele
 
 :::
 
+:::info
+
+An alternative to using prysmctl is calling the prysm api directly, making use of the `eth/v2/debug/beacon/states/{state_id}` and `eth/v2/beacon/blocks/{block_id}` endpoints.
```
Comment (mick-prysm) [inline website/docs/prysm-usage/checkpoint-sync.md:84]:
```suggestion
An alternative to using `prysmctl` is calling the Prysm API directly, making use of the `eth/v2/debug/beacon/states/{state_id}` and `eth/v2/beacon/blocks/{block_id}` endpoints.
```

**Comment by mick-prysm** [comment]
Thanks very much @muang0! You can ignore the CLA bot :)

**Comment by kasey** [comment]
Thank you so much for your contribution! Unfortunately, the process isn't quite that simple and requires code that understands how to work with ssz values to get the right values. The process to get the block goes like this:
1. download the latest finalized state (using the Beacon Node API you mention - `eth/v2/debug/beacon/states/finalized`
2. compute the root of the block used in that state, which is where the complexity comes in:
  a. compute the hash tree root of the retrieved `BeaconState`
  b. extract the `latest_block_header` from the retrieved `BeaconState` and fill in the `state_root` with the root computed in step 1
  c. compute the hash tree root of the `latest_block_header` with the `state_root` filled in
3. retrieve the block by the root computed in step 2c

That said, we have a release coming up very soon which will include binary releases of `prysmctl`. Hopefully that removes the concern of these complicated steps that require go compilation.

**Comment by mick-prysm** [comment]
Thanks Kasey. cc @muang0 - I'll close this + https://github.com/prysmaticlabs/documentation/issues/594. Thanks again for the contribution :)

**Pull request closed without merging**